apps/nextjs-website/src/components/atoms/ChatMessage/ChatMessage.tsx

```diff
@@ -0,0 +1,56 @@
+import { Box, Stack, Typography, useTheme } from '@mui/material';
+import PersonOutlineIcon from '@mui/icons-material/PersonOutline';
+import AdbOutlinedIcon from '@mui/icons-material/AdbOutlined';
+
+type ChatMessageProps = {
+  message: string;
+  sender?: string;
+  timestamp: string;
+};
+
+const ChatMessage = ({ message, sender, timestamp }: ChatMessageProps) => {
+  const { palette } = useTheme();
+  const bgColor = sender ? palette.background.paper : `${palette.info.light}80`;
```

Comment:
To be more clear instead of interpolation i suggest to use MUI's alpha function
```suggestion
  const bgColor = sender
    ? palette.background.paper
    : alpha(palette.info.light, 0.5);
```

Reply:
Nice ty